components/Alert/Alert.html

```diff
@@ -0,0 +1,157 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="author" content="Vrushabh Dhatrak" />
+    <meta name="description" content="The Alert component comes with different states of alert such as - error, warning, success and info." />
+    <link rel="stylesheet" href="./../../styles/main.css" />
+    <script src="./../../script/header.js" defer></script>
+    <script src="./../../script/navigation.js" defer></script>
+    <script src="https://code.iconify.design/2/2.1.1/iconify.min.js"></script>
```

Comment:
I had a doubt here, shouldn't script be added at the end of body, since you are not using defer here. Correct me if wrong

Reply:
No that file used for icon